lib/service/tracking/tracking_cubit.dart

```diff
@@ -16,3 +16,81 @@ extension LandingPageTracking on TrackingCubit {
     trackingRepository.track(LandingPageEvent.onPressedStartTrading());
   }
 }
+
+extension PoolPageTracking on TrackingCubit {
+  void onPoolCreated(
```

Comment:
There should be one extension / feature layer (Login/Trade/Pool/settings/...). 
I believe it should also keep a 1 on 1 parity with the event grouping.